content/TruenasApps/UsingCustomApp.md

```diff
@@ -48,138 +35,1199 @@ If you are unsure about any configuration settings, review the [Install Custom A
+
+   Select whether to mount the ixVolume as **Read Only** or to **Enable ACL** and configure settings as needed.
+   {{< /expand >}}
+
+   {{< expand "Host Path (Path that already exists on the system)" "v" >}}
+   To configure a persistent host path to an existing dataset, select **Host Path (Path that already exists on the system)**.
+   This is the recommended setting for on-disk storage in a production deployment.
 
-8. Enter any additional settings required for your application, such as workload details or adding container settings for your application.
+   {{< trueimage src="/images/SCALE/Apps/InstallCustomAppAddHostPathVol.png" alt="Host Path Settings" id="Host Path Settings" >}}
 
-9. Select the **Scaling/Upgrade Policy** to use.
-   The default is **Kill existing pods before creating new ones**.
+   Enter in **Mount Path** the <file>**path/to/directory**</file> where the host path mounts inside the container.
+   Then define the **Host Path**. Enter a path or click <span class="material-icons">arrow_right</span> to the left of <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 24 24"><path fill="currentColor" d="M5 21L3 9h18l-2 12zm5-6h4q.425 0 .713-.288T15 14t-.288-.712T14 13h-4q-.425 0-.712.288T9 14t.288.713T10 15M6 8q-.425 0-.712-.288T5 7t.288-.712T6 6h12q.425 0 .713.288T19 7t-.288.713T18 8zm2-3q-.425 0-.712-.288T7 4t.288-.712T8 3h8q.425 0 .713.288T17 4t-.288.713T16 5z"/></svg> **/mnt** to browse to the location of the dataset in TrueNAS.
 
-10. Use **Resource Reservation** to allocate GPU resources if available and required for the application.
+   Select whether to mount the host path as **Read Only** or to **Enable ACL** and configure settings as needed.
+   {{< /expand >}}
 
-11. Set any **Resource Limits** you want to impose on this application.
-    Select **Enable Pod resource limits** to display the **CPU Limit** and **Memory Limit** fields.
+   {{< expand "SMB/CIFS Share (Mounts a volume to a SMB share)" "v" >}}
+   To mount an existing SMB share as a Docker [volume](https://docs.docker.com/engine/storage/#volumes), select **SMB/CIFS Share (Mounts a volume to a SMB share)**.
 
-12. Enter or select any **Portal Configuration** settings to use.
-    Select **Enable WebUI Portal** to display UI portal settings.
+   {{< trueimage src="/images/SCALE/Apps/InstallCustomAppAddSMB.png" alt="SMB/CIFS Share Settings" id="SMB/CIFS Share Settings" >}}
 
-13. Click **Install** to deploy the container.
+   Enter in **Mount Path** the <file>**path/to/directory**</file> where the ixVolume mounts inside the container.
+
+   Enter in **Server** the IP address for the SMB server, for example *192.168.1.100*.
+   This can be the TrueNAS host.
+
+   Enter the name of the SMB share in **Path**, for example *my-share*.
+
+   Enter the **Username** and **Password** of an account with permission to access the SMB share.
```

Comment:
In the use case in these sentences we should not make the field the noun and use UI emphasis. Suggest the following changes:
```suggestion
   Enter the IP address for the SMB server in **Server**, for example *192.168.1.100*.
   This can be the TrueNAS host.

   Enter the name of the SMB share in **Path**, for example *my-share*.

   Enter the username and password of an account with permission to access the SMB share.
```